doc/source/cluster/aws-tips.rst

```diff
@@ -71,3 +71,202 @@ You should see something like
+    and must be manually provided in your alarm config file.
+
+   For more advanced CloudWatch configurations options, refer to `agent <https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Agent-Configuration-File-Details.html>`_, `dashboard <https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/CloudWatch-Dashboard-Body-Structure.html>`_ and `alarm <https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_PutMetricAlarm.html>`_ AWS documentations.
+
+   CloudWatch Agent, Dashboard, and Alarm JSON config files support the following variables:
+    {instance_id}: Replaced with each EC2 instance ID in your Ray cluster.
+
+    {region}: Replaced with your Ray cluster's region.
+
+    {cluster_name}: Replaced with your Ray cluster name.
+
+    See CloudWatch Agent `Configuration File Details <https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Agent-Configuration-File-Details.html>`_ for additional variables supported natively by the Unified CloudWatch Agent.
+
+.. note::
+    For CloudWatch alarm JSON configuration file, remember to add your alarm actions as shown below.
+
+.. code-block:: json
+
+     "AlarmActions":[
+         "TODO: Add alarm actions! See https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/AlarmThatSendsEmail.html"
+      ]
+
+3. Reference your own CloudWatch JSON configuration files as file path in YAML cluster config file.
+
+   You need to specify the file path to CloudWatch JSON configuration files relative to YAML cluster launcher config file like:
+
+.. code-block:: yaml
+
+    agent:
+        config: "cloudwatch/example-cloudwatch-agent-config.json"
+
+4. Setup minimal IAM permission required for CloudWatch.
+
+   As for IAM role, ``ray-autoscaler-cloudwatch-v1`` is the default role for ensuring Unified CloudWatch Agent has permission to perform CloudWatch related operations.
+   A corresponding instance profile is created.
+   Remember to use this default ``ray-autoscaler-cloudwatch-v1`` instance profile for worker nodes in YAML cluster config file to ensure ray worker nodes have the right permissions.
+
+.. code-block:: yaml
+
+    ray.worker.default:
+        node_config:
+            InstanceType: c5a.large
+            IamInstanceProfile:
+                Name: ray-autoscaler-cloudwatch-v1
+
+5. To ship Ray system-level Prometheus metrics to CloudWatch.
+
+   If you want to export Ray's Prometheus system metrics to CloudWatch, you should first ensure that your cluster has the
+   Ray Dashboard installed, then uncomment the ``head_setup_commands`` section in `example-cloudwatch.yaml file <https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/aws/example-cloudwatch.yaml>`_ file.
+   You can find Ray Prometheus metrics in the ``{cluster_name}-ray-prometheus`` metric namespace.
+
+.. code-block:: yaml
+
+    head_setup_commands:
```

Comment:
This is feeling pretty difficult to read/understand. Can we break it up into more commands, or use 

```
- bash -c "
mutli-line;
command;"
``` 

or something?